lib/hackmode/realmem.test.js

```diff
@@ -1,30 +1,112 @@
 const { expect } = require('chai')
 
 const RealMem = require('./realmem').RealMem
+const getLoc = require('./realmem').getLoc
 
-describe('HackMode Mem Suite', () => {
+describe('RealMem Suite', () => {
   let m
 
   beforeEach(() => {
     m = new RealMem()
     m.insert({
-      address: 'test1'
+      entryAddress: 'entryAddress1',
+      address: 'aspectAddress1'
     })
     m.insert({
-      address: 'test2'
+      entryAddress: 'entryAddress2',
+      address: 'aspectAddress2'
+    })
+    m.insert({
+      entryAddress: 'entryAddress2',
+      address: 'aspectAddress3'
     })
   })
 
   it('should be a function', () => {
     expect(typeof RealMem).equals('function')
   })
 
-  it('should get', () => {
-    expect(m.get('test1')).deep.equals({
-      entry: {
-        address: 'test1'
-      },
-      meta: []
+  it('should get entryAddress1', () => {
+    let e1 = m.get('entryAddress1')
+    expect(e1).deep.equals([{
+      address: 'aspectAddress1',
+      json: JSON.stringify({
+        entryAddress: 'entryAddress1',
+        address: 'aspectAddress1'
+      })
+    }])
+  })
+
+  it('should get entryAddress2', () => {
+    let e1 = m.get('entryAddress2')
+    expect(e1).deep.equals([{
+      address: 'aspectAddress2',
+      json: JSON.stringify({
+        entryAddress: 'entryAddress2',
+        address: 'aspectAddress2'
+      })
+    }, {
+      address: 'aspectAddress3',
+      json: JSON.stringify({
+        entryAddress: 'entryAddress2',
+        address: 'aspectAddress3'
+      })
+    }
+    ])
+  })
+
+  it('should have all entryAddress', () => {
+    expect(m.has('entryAddress1', 'aspectAddress1')).equals(true)
+    expect(m.has('entryAddress2', 'aspectAddress1')).equals(false)
+    expect(m.has('entryAddress2', 'aspectAddress2')).equals(true)
+    expect(m.has('entryAddress2', 'aspectAddress3')).equals(true)
+    expect(m.has('entryAddress2', 'aspectAddress1')).equals(false)
```

Comment:
This line is redundant with line 60 above. Consider:
`    expect(m.has('entryAddress1', 'aspectAddress2')).equals(false)`
`    expect(m.has('entryAddress1', 'aspectAddress3')).equals(false)`

Reply:
👍